django/contrib/admin/static/admin/css/dark_mode.css

```
@@ -30,6 +30,10 @@
       --close-button-bg: #333333;
       --close-button-hover-bg: #666666;
     }
+
+    :not(html[data-theme="light"]) > body {
+        color-scheme: dark;
+    }
```

Same here, we can get the same effect setting this on `:root` just like with all other color changes.


```suggestion
      --close-button-hover-bg: #666666;
      
      color-scheme: dark;
    }
```